libs/lwt_platform/Lwt_platform.mli

```diff
@@ -13,3 +13,6 @@ val set_engine : unit -> unit
     This is important as select/poll is not great, and can easily
     run out of FDs, crashing the LS.
   *)
+
+val sleep : float -> unit Lwt.t
+(** [sleep delay] returns a promise that will be resolved after [delay] seconds. *)
```

Comment:
Is `Lwt_unix.sleep` not suitable?

Please explain what this module achieves and why it should be used or not used. The comment at the beginning of the file `(* See this modules dune comment for why this exists *)` is not helpful because (1) it takes time to locate and open the dune file and (2) said dune file doesn't give the promised explanation 😐

Reply:
I'll add a short explanation and fix this comment so it points to the right dune file (it's the next level up) for further explanation

Reply:
Oh wait it does point to the right dune file :/ I'll add a short explanation